docs/concepts/flows.md

```diff
@@ -7,121 +7,100 @@ tags:
-| `task_runner`                                      | An optional [task runner](/concepts/task-runners/) to use for task execution within the flow when you `.submit()` tasks. If not provided and you `.submit()` tasks, the `ConcurrentTaskRunner` will be used.         |
-| `timeout_seconds`                                  | An optional number of seconds indicating a maximum runtime for the flow. If the flow exceeds this runtime, it will be marked as failed. Flow execution may continue until the next task is called.                   |
-| `validate_parameters`                              | Boolean indicating whether parameters passed to flows are validated by Pydantic. Default is `True`.                                                                                                                  |
-| `version`                                          | An optional version string for the flow. If not provided, we will attempt to create a version string as a hash of the file containing the wrapped function. If the file cannot be located, the version will be null. |
-
-For example, you can provide a `name` value for the flow. Here we've also used the optional `description` argument and specified a non-default task runner.
+| `description`                                      | Optional string description for the flow. Defaults to the docstring of the decorated function.|
+| `name`                                             | Optional name for the flow. Defaults to the function name.|
+| `retries`                                          | Optional number of times to retry on flow run failure.|
+| <span class="no-wrap">`retry_delay_seconds`</span> | Optional delay (in seconds) between retries after a flow run failure. Applicable only if `retries` is set.|
+| `flow_run_name`                                    | Optional template string to define the name for flow runs. Can reference flow parameters or use a function. For example, you can use the `{name}-on-{date:%A}` value for dynamic flow run naming|
+| `task_runner`                                      | Optional task runner to use for task execution within the flow. Defaults to `ConcurrentTaskRunner`.|
+| `timeout_seconds`                                  | Optional maximum runtime (in seconds) for the flow. If exceeded, the flow is marked as failed.|
+| `validate_parameters`                              | Boolean indicating whether to validate flow parameters using Pydantic (`default: True`). Where possible, Prefect forces values into the correct type. For example, if you define a parameter as `x: int` and "5" is passed, it resolves to `5`.|
+| `version`                                          | Optional version string for the flow. Defaults to a hash of the file containing the wrapped function. If the file cannot be found, the `version` value is `null`. |
+
+### Examples
+The following examples show how to customize certain flow behaviors.
```

Comment:
I interpret "behavior" to mean the way that my flow runs. While the task runner changes the flow's behavior by the definition, the description and flow_run_name are not so much behavior changes as they are record keeping changes.